app/presenters/organisations/contacts_presenter.rb

```diff
@@ -0,0 +1,105 @@
+module Organisations
+  class ContactsPresenter
+    include ActionView::Helpers::TagHelper
+    include ActionView::Helpers::UrlHelper
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def has_contacts?
+      @org.ordered_contacts && @org.ordered_contacts.length.positive?
+    end
+
+    def contacts
+      format_contacts(@org.ordered_contacts)
+    end
+
+    def has_foi_contacts?
+      true if @org.foi_contacts && @org.foi_contacts.length.positive?
```

Comment:
Minor: do we need `true if` here? (and yes, this is code you've moved so it's not on you, but I just noticed it in comparison to the `has_contacts?` method above).

Reply:
Good spot, probably not. I'll try removing 👍